### PR TITLE
Update the build status badge to GH Actions

### LINKED ISSUE #329: Change the build status badge in the readme to Github Actions instead of Azure Pipelines
**Is your feature request related to a problem? Please describe.**
In the readme, the build status badge refers to the Azure Pipelines, I would like it to refer to the Github Actions pipeline

**Describe the solution you'd like**
In the readme, the build status badge refers to the Azure Pipelines, I would like it to refer to the Github Actions pipeline

**Describe alternatives you've considered**
nothing

**Additional context**
billede

### PR DESCRIPTION
**Pull request by mathiasandresen**
closes #329

The badge is taken directly from the GitHub Actions pane "Create status badge"

### CODE REVIEW

**Review by mads-bo — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ When creating a new branch for this project. One should stay on the Weekplanner-
 
 GNU GENERAL PUBLIC LICENSE V3
 
-Copyright [2018] [Aalborg University]
\ No newline at end of file
+Copyright [2018] [Aalborg University]
```
Comment (Lortemanden) [inline README.md]: Should copyright not be changed to 2020?
image

Comment (mathiasandresen) [reply README.md]: probably, but that isn't related to this issue. I don't know why it says its changed, since I didn't touch it.

**Review by Semicolon1071 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mathiasandresen) [reply README.md]: I can change it if you want?

Comment (Lortemanden) [reply README.md]: It seems small to make a issue for. Just change it quick and then it's all good

Comment (mathiasandresen) [reply README.md]: done

**Review by Lortemanden — APPROVED** [review]
Looks good

**Review by mads-bo — APPROVED** [review]

**Comment by mathiasandresen** [comment]
@Andreas2762 wanna re-review it? :)

**Review by Semicolon1071 — APPROVED** [review]

**Pull request merged**